tools/StaticAnalysis/Exceptions/Az.ADDomainServices/SignatureIssues.csv

```diff
@@ -10,3 +10,5 @@
 "Az.ADDomainServices","Update-AzADDomainService","Update-AzADDomainService","1","8100","Update-AzADDomainService Does not support ShouldProcess but the cmdlet verb Update indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
```

Comment:
Why we changed BreakingChangeIssues.csv, SignatureIssues.csv and validateHelpIssues.csv?

Reply:
Removed.